pico/tests/resources-pico/src/main/java/io/helidon/pico/tests/pico/interceptor/XImpl.java

```diff
@@ -30,7 +30,7 @@
 import jakarta.inject.Singleton;
 
 /**
- * This test case is applying {@link InterceptorBasedAnno} (an {@link io.helidon.pico.InterceptedTrigger}) using the no-arg
+ * This test case is applying {@link InterceptorBasedAnno} (an {@link io.helidon.pico.pi.InterceptedTrigger}) using the no-arg
```

Comment:
typo - `.pi.` instead of `.api.`

Reply:
interesting how checkstyles / spotbugs didn't pick that up.

Reply:
pushed